README.md

```diff
@@ -1,5 +1,27 @@
 # battleship
```

Comment:
thanks for getting this started! 

Taking a look and will add/ edit and format:)